src/Uno.UI/UI/Xaml/Media/Imaging/BitmapImage.Cache.skia.cs

```diff
@@ -97,30 +93,44 @@ public static bool TryGetFromUri(Uri uri, out Task<ImageData>? task)
 				}
 			}
 
+			public static void Remove(Uri uri, Task<ImageData> task)
```

Comment:
Let's keep internal on `BitmapImage`:

```suggestion
			internal static void Remove(Uri uri, Task<ImageData> task)
```